evap/templates/base.html

```diff
@@ -160,6 +160,37 @@
         {% endblock %}
     </div>
 </div>
+
+{% if user.is_authenticated and not user.is_staff %}
```

Comment:
the modal is currently always visible on the bottom of the page. please hide it until someone clicks the button ;)
also, please show the user's email address in the modal so that the user knows, that it will be connected to the message (use a disabled form field).